clar2wasm/src/words/principal.rs

```diff
@@ -306,6 +307,7 @@ mod tests {
     //- is-standard
 
     #[test]
+    #[cfg(not(feature = "test-clarity-v1"))]
```

Comment:
it's a bit too verbose to have this in front of every test, better do them on a module level like above.

Reply:
Refactored the code to use modules. 👍